### ISSUE TITLE
Add counter to show how many files have been completed

### ISSUE THREAD

**Issue by lucasbento** [react-native-community/upgrade-helper#30]
# Feature Request

Idea initiated in #21, basically add a counter like `2/10` in the bottom right of the page to show how many files have been completed and how many are still left, the counter should show only after you complete at least one file.

**Comment by pvinis** [comment]
One other thing for this that might help, is to put the "done" files on the bottom of the list.

**Comment by KevinHu2014** [comment]
Could I take this one?

**Comment by lucasbento** [comment]
Hi @KevinHu2014, I'm glad to see that you are interested on contributing!

Unfortunately I'm already working on this for a while, I'll probably get a PR opened by Monday, would appreciate if you could review and test it though.

This one that @pvinis mentioned is actually up for grabs:
> One other thing for this that might help, is to put the "done" files on the bottom of the list.

I didn't focus on it because I'm not sure of how to smoothly transition the files to the bottom, would you be interested in working on it? I can definitely help you out with it :)

**Comment by KevinHu2014** [comment]
@lucasbento  Sure, I'm happy to help :) 

By the way I'm thinking about another approach on this

> One other thing for this that might help, is to put the "done" files on the bottom of the list.

How about after hitting the complete button, the page autoscroll down one section (or block? not sure what's best word for this). 
Due to, if the user regrets on checking the button (or wants to see the previous complete one's) , user have to scroll all the way down to the bottom of the list to see the info. Times like this might felt a little annoying😅

@pvinis  What do you think ?

**Comment by pvinis** [comment]
I think moving the file might be better, just for the case where (at least I do that) someone scrolls up and down to see what is left, and often "similar" files are scattered, like some ios files might have some js file changes in between.

So I feel like if we only autoscroll, then second the user scrolls, we have lost what's done and we have to keep looking at the tiny blue checkmark. Whereas if the actual list item (file/diff) moves on the bottom, maybe even below a separator, it becomes pretty clear what is done and what is not done yet.

Does that make sense? I feel like this case is more important than an accidental click of the done button as our main usecases.

**Comment by KevinHu2014** [comment]
> So I feel like if we only autoscroll, then second the user scrolls, we have lost what's done and we have to keep looking at the tiny blue checkmark.

Cool, that totally make sense. 👍🏻 Going through files often makes user lose track of it, and moving file to the bottom will definitely be more clear for what has been done.

I'll try my best to make this happen with a smoothly transition and add a separator below.

**Comment by lucasbento** [comment]
@KevinHu2014: awesome, please let me know if you need any kind of help!

Shall I create a separate issue for this?

**Comment by lucasbento** [comment]
Created #43 for this :)

**Comment by lucasbento** [comment]
Done in #44.
